internal/network/ntp_utils.go

```diff
@@ -72,13 +72,13 @@ func createChronyManifestContent(c *common.Cluster) (string, error) {
 	sources := make([]string, 0)
 
 	for _, host := range c.Hosts {
-		if swag.StringValue(host.Status) == models.HostStatusDisabled {
+		if swag.StringValue(host.Status) == models.HostStatusDisabled || host.NtpSources == "" {
```

Comment:
Is it expected? No warning about it?

Reply:
At the moment we decided that NTP shouldn't fail or block an installation.
Anyhow NTP sources validations shouldn't be in here but in the Validator modules